<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>authorization</class>
 <widget class="QMainWindow" name="authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Authorization</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_about_authorization">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;Do you want to login or register?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_authorization">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Account authorization&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_registration">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>Registration</string>
    </property>
   </widget>
   <widget class="QLabel" name="application_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>581</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Diary of completing sports tasks&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>415</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: lightblue;</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_image_authorization">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Icons/authorization.gif</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="CB_languages">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>415</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Russian</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
